lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -503,19 +491,7 @@ export class PublicClientApplication {
      * @returns {string} post logout redirect URL
      */
     public getPostLogoutRedirectUri(): string {
-        if (this.config.auth.postLogoutRedirectUri) {
-            if (typeof this.config.auth.postLogoutRedirectUri === "function") {
-                const postLogoutUri = this.config.auth.postLogoutRedirectUri();
-                if (StringUtils.isEmpty(postLogoutUri)) {
-                    throw BrowserConfigurationAuthError.createPostLogoutRedirectUriEmptyError();
-                }
-                return postLogoutUri;
-            }
-
-            return this.config.auth.postLogoutRedirectUri;
-        }
-        // This should never throw unless window.location.href is returning empty.
-        throw BrowserConfigurationAuthError.createPostLogoutRedirectUriEmptyError();
+        return this.config.auth.postLogoutRedirectUri ? this.config.auth.postLogoutRedirectUri : BrowserUtils.getCurrentUri();
```

Comment:
These can just be `||`: `return this.config.auth.postLogoutRedirectUri || BrowserUtils.getCurrentUti();`

Reply:
Done